setup.cfg

```diff
@@ -28,4 +28,5 @@ python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
 [options.entry_points]
 console_scripts =
     validate_secrets_are_encrypted = pre_commit_hooks.validate_secrets_are_encrypted:main
-    validate_encrypted_secret_name = pre_commit_hooks.validate_encrypted_secret_name:main
\ No newline at end of file
+    validate_encrypted_secret_name = pre_commit_hooks.validate_encrypted_secret_name:main
+    check_notebook_has_parameters_cell = pre_commit_hooks.check_notebook_has_parameters_cell:main
```

Comment:
add a new line

Reply:
> thought you tested the last PR before merging?

I thought it ran the tests and passed but they never ran